scripts/portstat

```diff
@@ -70,7 +71,14 @@ RateStats = namedtuple("RateStats", ratestat_fields)
 The order and count of statistics mentioned below needs to be in sync with the values in portstat script
 So, any fields added/deleted in here should be reflected in portstat script also
 """
-BUCKET_NUM = 45
+BUCKET_NUM = 49
+
+wred_green_pkt_stat_capable = "False"
```

Comment:
Let's use `"false"` to be more consistent with [swss](https://github.com/sonic-net/sonic-swss/pull/2750/files#diff-bb736c6a11da68afacf95d761fb52a3c11726cdd72c967d4803b8c9b1a6dcb69R937) and below `"true"`.

Reply:
Agree. I have addressed the review comment..!